src/app/app.component.ts

```diff
@@ -1,16 +1,36 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterOutlet } from '@angular/router';
-import { LetterCardComponent } from './components/letter-card/letter-card.component';
 import { GridComponent } from './components/grid/grid.component';
+import { HttpClient } from '@angular/common/http';
+import { GameStateService } from './services/game-state.service';
+
+const API_URL = 'https://random-word-api.herokuapp.com/word?lang=es&';
 
 @Component({
   selector: 'app-root',
   standalone: true,
   imports: [CommonModule, RouterOutlet, GridComponent],
   templateUrl: './app.component.html',
-  styleUrl: './app.component.scss'
+  styleUrl: './app.component.scss',
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'wordle-angular';
+
+  constructor(
+    private readonly httpClient: HttpClient,
+    private gameStateServide: GameStateService
+  ) {}
+
+  ngOnInit(): void {
+    this.httpClient.get<string>(API_URL + 'length=5').subscribe((item) => {
+      this.gameStateServide.secretWord = item.toString();
+      this.gameStateServide.secretWord = this.gameStateServide.secretWord
+        .normalize('NFD')
+        .replace(/[\u0300-\u036f]/g, '')
+        .toLowerCase();
```

Comment:
Todas las transformaciones las puedes hacer antes de que el observable emita sus valores. Es decir, antes de que el suscribe envie data. 

Vamos a hacer uso de la función pipe.

this.httpClient.get<string>(API_URL + 'length=5').pipe((item) => 
  item.toString().normalize('NFD').replace(/[\u0300-\u036f]/g, '').toLowerCase();
)